uint16[3] sonar_cm

float32 new_t
